### PR TITLE
Arrow Update and CI Fix

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

File: datatypes/src/collections/feature_collection.rs [hunk 1]
```diff
@@ -1161,7 +1137,7 @@ mod tests {
         for i in 0..10 {
             assert_eq!(
                 gen_collection(i).byte_size(),
-                empty_hash_map_size + struct_stack_size + 264 + time_interval_size(i)
+                empty_hash_map_size + struct_stack_size + 192 + time_interval_size(i)
```
Comment (jdroenner) [inline datatypes/src/collections/feature_collection.rs:1140]: why did this magic number change?

File: operators/src/mock/mock_feature_collection_source.rs [hunk 2]
```diff
@@ -141,7 +141,7 @@ mod tests {
 
         let serialized = serde_json::to_string(&source).unwrap();
         let collection_bytes = [
-            65, 82, 82, 79, 87, 49, 0, 0, 144, 1, 0, 0, 16, 0, 0, 0, 0, 0, 10, 0, 14, 0, 12, 0, 11,
+            65, 82, 82, 79, 87, 49, 0, 0, 148, 1, 0, 0, 16, 0, 0, 0, 0, 0, 10, 0, 14, 0, 12, 0, 11,
```
Comment (jdroenner) [inline operators/src/mock/mock_feature_collection_source.rs:144]: is there a new test dataset?

**Review by jdroenner — COMMENTED** [review]
looks fine. just some questions from my side.

File: datatypes/src/collections/feature_collection.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply datatypes/src/collections/feature_collection.rs:1140]: It seems like the calculation method changed, I copy-pasted it from their master branch.

File: operators/src/mock/mock_feature_collection_source.rs (hunk 2 above)
Comment (ChristianBeilschmidt) [reply operators/src/mock/mock_feature_collection_source.rs:144]: No, but this is the flight format. I was also curious why this changed, but the byte representation is very suboptimal for now. In the future, we should use the json writer if it is available.

File: datatypes/src/collections/feature_collection.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply datatypes/src/collections/feature_collection.rs]: 76b4180

**Pull request merged**